examples/panel/weights-and-biases-llm/requirements.txt

```diff
@@ -2,4 +2,9 @@ openai==1.12.0
 panel==1.3.8
 ploomber-cloud
 wandb==0.16.3
-ipython
\ No newline at end of file
+ipython
+selenium
+seaborn
+ipywidgets-bokeh
+
+#conda install -c conda-forge firefox geckodriver
```

Comment:
remove this conda command

Reply:
resolved https://github.com/ploomber/doc/pull/124/commits/5e267ebaed41fdcbfe65eb6bbf160df00156605b